types/office-js/index.d.ts

```diff
@@ -16408,6 +16423,33 @@ declare namespace Office {
         setStartTime(time: string): void;
     }
 
+    /**
+     * Represents the sharedProperties of an appointment or message in an Outlook add-in.
```

Comment:
Here especially the term "sharedProperties" should be described in more descriptive terms, since this is the ultimate reference line about this object.